whitesource/util.py

```diff
@@ -247,6 +248,9 @@ def scan_artifact_with_ws(
     requester_mail: str,
```

Comment:
btw: with `ws` being used both for websocket, and whitesource, we now have a naming-clash.

--> maybe rather rename to whitesource and websocket (or some abbrevs, such as websock, white_src, .. it is ok to be brief, but we should avoid ambiguities)

Reply:
Didn't even notice, will go for **ws** and **white_src** 👍